src/main/java/uk/ac/ebi/eva/pipeline/io/mappers/VariantVcfFactory.java

```diff
@@ -296,6 +304,29 @@ private String processGenotypeField(int alternateAlleleIdx, String genotype) {
         return genotype.intern();
     }
 
+    protected void checkForNonVariantSites(Variant variant, String fileId, String studyId) throws VCFRecordWithNonVariantSites {
```

Comment:
I'm quite bad at names, but I think "sites" is misleading here. what about `checkActualVariant`?